manifest.json

```diff
@@ -1,28 +1,47 @@
 {
-  "name": "@deskpro-apps/my-app",
-  "title": "My App",
-  "description": "My Deskpro app",
+  "name": "@deskpro-apps/copper",
+  "title": "Copper",
+  "description": "Elevate your customer relationships, optimize your sales pipeline, and simplify your business workflows with Copper.",
   "version": "0.0.1",
   "scope": "agent",
   "isSingleInstall": false,
   "hasDevMode": true,
-  "targets": [{ "target": "ticket_sidebar", "entrypoint": "index.html" }],
+  "targets": [
+    {
+      "target": "user_sidebar",
+      "entrypoint": "index.html"
+    }
+  ],
   "settings": {
-    "example": {
-      "title": "Example Setting",
-      "description": "Example backend only setting",
+    "api_key": {
+      "title": "Api Key",
+      "description": "API Key can be obtained by following our setup guide",
+      "type": "string",
+      "isRequired": true,
+      "isBackendOnly": true,
+      "order": 10
+    },
+    "api_key_owner_email": {
+      "title": "API Key Owner Email",
+      "description": "The email address associated with the account that the API Key belongs to.",
       "type": "string",
-      "isRequired": false,
-      "isBackendOnly": true
+      "isRequired": true,
+      "isBackendOnly": true,
+      "order": 20
     }
```

Comment:
here we need to verify the settings for better UX for our clients.

as an example, we have this in Youtrack App:
https://github.com/DeskproApps/youtrack/blob/master/manifest.json#L50
https://github.com/DeskproApps/youtrack/blob/master/src/pages/VerifySettings/VerifySettings.tsx